Changelog.md

```diff
@@ -1,4 +1,5 @@
 # Next
+- Added tests for Single+Response and refactored Shared Test Helpers.
```

Comment:
I don't mind removing this, I just saw a line saying 
"Moved tests to Moya.xcodeproj." so wasn't sure 

Should I move it to the bottom of breaking or get rid of it completely ?